watertap/property_models/tests/test_coag_prop_pack.py

```diff
@@ -0,0 +1,204 @@
+        #       all will now be built
+
+    @pytest.mark.unit
+    def test_stats(self, coag_obj):
+        model = coag_obj
+
+        # Check to make sure we have the correct DOF and
+        #   check to make sure the units are correct
+        assert degrees_of_freedom(model) == 6
+        assert_units_consistent(model)
+
+        # Fix some variables and verify it closes the system of equations
+        model.fs.stream[0].temperature.fix(298)
+        model.fs.stream[0].pressure.fix(101325)
+        model.fs.stream[0].flow_mass_phase_comp['Liq', 'H2O'].fix(1)
+        model.fs.stream[0].flow_mass_phase_comp['Liq', 'TSS'].fix(0.01)
+        model.fs.stream[0].flow_mass_phase_comp['Liq', 'TDS'].fix(0.01)
+        model.fs.stream[0].flow_mass_phase_comp['Sol', 'Sludge'].fix(0.001)
+        assert degrees_of_freedom(model) == 0
+
+    @pytest.mark.unit
+    def test_scaling(self, coag_obj):
+        model = coag_obj
+
+        # Set some scaling factors and look for 'bad' scaling
+        model.fs.properties.set_default_scaling('flow_mass_phase_comp', 1, index=('Liq', 'H2O'))
+        model.fs.properties.set_default_scaling('flow_mass_phase_comp', 1e2, index=('Liq', 'TSS'))
+        model.fs.properties.set_default_scaling('flow_mass_phase_comp', 1e2, index=('Liq', 'TDS'))
+        model.fs.properties.set_default_scaling('flow_mass_phase_comp', 1e3, index=('Sol', 'Sludge'))
+        iscale.calculate_scaling_factors(model.fs)
+
+        # check that all variables have scaling factors
+        unscaled_var_list = list(iscale.unscaled_variables_generator(model))
+        assert len(unscaled_var_list) == 0
+
+        # check that all constraints have been scaled
+        unscaled_constraint_list = list(iscale.unscaled_constraints_generator(model))
+        assert len(unscaled_constraint_list) == 0
+
+        # check if any variables are badly scaled
+        badly_scaled_var_list = list(iscale.badly_scaled_var_generator(model, large=1e2, small=1e-2))
+        if len(badly_scaled_var_list) != 0:
+            lst = []
+            for (var, val) in badly_scaled_var_list:
+                lst.append((var.name, val))
+                print(var.name, var.value)
+            print("The following variable(s) are poorly scaled: {lst}".format(lst=lst))
+        assert len(badly_scaled_var_list) == 0
+
+    @pytest.mark.component
+    def test_initialization(self, coag_obj):
+        model = coag_obj
+
+        # call the 'calculate_state', which will call initialize and return results
+        #       pass var_args as the fixed states from 'test_stats'
+        args = {('temperature', None): 298,
+                ('pressure', None): 101325,
+                ('flow_mass_phase_comp', ('Liq','H2O')): 1,
+                ('flow_mass_phase_comp', ('Liq','TDS')): 0.01,
+                ('flow_mass_phase_comp', ('Liq','TSS')): 0.01,
+                ('flow_mass_phase_comp', ('Sol','Sludge')): 0.001}
+        results = model.fs.stream.calculate_state(var_args=args)
```

Comment:
We usually just use `initialize` or `initialization_tester` for this test

Reply:
I know, but doing this increases test coverage and accomplishes the same thing.